src/node/desktop/test/core/err.test.ts

```diff
@@ -26,16 +26,15 @@ function beUnsuccessful(): Err {
   return new Error('Some error');
 }
 
-describe('Err', () => {
+describe('Err Using Chai Asserts', () => {
```

Comment:
Think this can just be left as-is, versus mentioning 'Chai Asserts'.

Reply:
Oops, leftover from when I was comparing chai vs node asserts. Great catch